monosi/monitors/table_metrics.py

```diff
@@ -13,8 +13,7 @@ class TableMetricsMonitor(Monitor):
     timestamp_field: str = field(default_factory=str)
     columns: Optional[List[str]] = field(default_factory=list)
     metrics: Optional[List[str]] = field(default_factory=list)
-    # timestamp_field_expression: str = ''
-    # where: str = ''
+    where: str = field(default_factory=str)
```

Comment:
custom filter input? does anything happen with it except that now we accept it as a valid argument?

Reply:
No - this is a good catch - technically I should have left 'where' commented as it's not used at the moment.